Library/Homebrew/download_strategy.rb

```diff
@@ -830,6 +838,10 @@ def stage
     safe_system(*args)
   end
 
+  def source_modified_time
+    Time.parse Utils.popen_read("fossil", "info", "tip", "-R", cached_location.to_s)[/^parent: *\h* (.*)$/, 1]
```

Comment:
- I think it would be good to make the pattern `^parent: +\h+ (.+)$` instead of `^parent: *\h* (.*)$`. Judging from the example output it seems reasonable to expect that there is always whitespace after the colon, always a hexadecimal ID, and always a non-empty date/time.
- Looking at the example output you provided makes me think we should be parsing the `uuid:` line instead of the `parent:` line. I tried to confirm that by searching Fossil's online documentation, but failed to find a definitive answer. Can you check that and/or point me at the relevant documentation?